annular-eclipse-2023/src/AnnularEclipse2023.vue

```diff
@@ -5,183 +5,208 @@
+                :color="accentColor"
+                :focus-color="accentColor"
+                :tooltip-text="showTextSheet ? 'Hide Info' : 'Learn More'"
+                :tooltip-location="'bottom'"
+              ></icon-button>
+              <icon-button
+                v-model="showVideoSheet"
+                fa-icon="video"
+                :color="accentColor"
+                :focus-color="accentColor"
+                tooltip-text="Watch video"
+                :tooltip-location="'bottom'"
+              >
+              </icon-button>
+              <icon-button
+                fa-icon="sun"
+                :color="accentColor"
+                :focus-color="accentColor"
+                tooltip-text="Center view on Sun"
+                :tooltip-location="'bottom'"
+                @activate="() => trackSun()"
+              >
+              </icon-button>
+              <icon-button
+                fa-icon="question"
+                :color="accentColor"
+                :focus-color="accentColor"
+                :tooltip-location="'bottom'"
+                @activate="() => { inIntro=true; introSlide = 2 }"
+              ></icon-button>
             </div>
-            
-          </div>
-
-          <div v-if="selectedLocation === 'User Selected'">
-            <icon-button
-              id="share"
-              fa-icon="share-nodes"
-              :color="accentColor"
-              :focus-color="accentColor"
-              background-color="transparent"
-              :box-shadow="false"
-              @activate="copyShareURL"
-            ></icon-button>
-          </div>
-          
+          </v-col>
+        </v-row>
+      </v-col>
+      <v-col cols="6" id="map-column">
+        <div id="map-container" >
+          <location-selector
+            v-if="learnerPath == 'Explore'"
+            :model-value="locationDeg"
+            @place="(place: typeof places[number]) => updateLocation(place.name)"
+            :detect-location="false"
+            :map-options="mapOptions"
+            :places="places"
+            :initial-place="places.find(p => p.name === 'selectedLocation')"
+            :place-circle-options="placeCircleOptions"
+            :selected-circle-options="selectedCircleOptions"
+            :selectable="false"
+            class="leaflet-map"
+          ></location-selector>
+
+          <span v-if="learnerPath=='Answer'">
+            <img alt="CosmicDS Logo" src="../../assets/EclipseMapPaths.png"/>
+          </span>
+
+          <location-selector
+            v-if="learnerPath == 'Choose'"
+            :model-value="locationDeg"
+            @update:modelValue="updateLocationFromMap"
+            :detect-location="false"
+            :selected-circle-options="selectedCircleOptions"
+            class="leaflet-map"
+          ></location-selector>
         </div>
       </v-col>
     </v-row>
-  </v-card>
+  </v-container>
+  <div v-if="!showGuidedContent" id="closed-top-container">
```

Comment:
The reason resizing isn't working is because the top bar is now a different element than the `onResize` function is currently looking at. The `updateGuidedContentHeight` should be updated to to look at the appropriate `id` given the value of `showGuidedContent`. 

replacing 
```
const guidedContentContainer = document.getElementById('guided-content-container') as HTMLElement;
```
with
```
let guidedContentContainer = null as HTMLElement | null;
      
if (this.showGuidedContent) { 
  guidedContentContainer = document.getElementById('guided-content-container') as HTMLElement;
} else {
  guidedContentContainer = document.getElementById('closed-top-container') as HTMLElement;
}
```

Should fix this
```suggestion
  <div v-if="!showGuidedContent" id="closed-top-container">
```